development-vm/replication/replicate-data-local.sh

```diff
@@ -7,6 +7,39 @@ set -eu
 
 . $(dirname $0)/common-args.sh
 
+if ! [ -x "$(command -v jq)" ]; then
+  echo 'Error: jq is not installed. https://stedolan.github.io/jq/' >&2
+  exit 1
+fi
+
+# setup AWS access
+SESSION_NAME=$(whoami)-$(date +%d-%m-%y_%H-%M)
+ROLE_ARN=$(awk '/role_arn/ {print $3}' ~/.aws/config)
+MFA_ARN=$(awk '/mfa_serial/ {print $3}' ~/.aws/config)
+
+if [[ "${MFA_TOKEN}" == "" ]]; then
+  echo "Error: must provide token"
+  echo "MFA_TOKEN=token $0"
+  exit 1
+fi
+
+unset AWS_SESSION_TOKEN
+
+CREDENTIALS=$(aws sts assume-role \
+                --role-session-name $SESSION_NAME \
+                --role-arn $ROLE_ARN \
+                --serial-number $MFA_ARN \
+                --token-code $MFA_TOKEN)
+
+if [[ $? == 0 ]]; then
+  echo "export AWS_ACCESS_KEY_ID=$(echo ${CREDENTIALS} | jq -r '.Credentials | .AccessKeyId')"
+  echo "export AWS_SECRET_ACCESS_KEY=$(echo ${CREDENTIALS} | jq -r '.Credentials | .SecretAccessKey')"
```

Comment:
Should we be logging keys like this, even in dev?

Reply:
It only exports them to the screen, they are only valid for an hour, it is quite useful as you can use them to log in manually if required.

Happy to remove if you feel strongly.

Reply:
so it has:

```
content_store_production	13.947265625GB
draft_content_store_production	15.9462890625GB
```

which are in the other dumps.